src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public Ball getRandomBall() {
+
+        Ball ball = new Ball();
+
+        ball.setBallColor(ColorSupplier.getRandomColor());
+
+        ball.setBallNumber(new Random().nextInt(100));
+
+        return ball;
+    }
+}
```

Comment:
1) It doesn't make any sense to make a Lottery extends ColorSupplier, especially in case you still want to go ahead with static getRandomColor method. 
But I would say that the best way here is to declare ColorSupplier as a field of Lottery class and inject in using constructor. And you definitely need to make getRandomColor() non-static as I already mentioned before.

2) There are also a few extra empty lines which can be confusing, because usually empty lines are used to break up pieces of code with related logic to improve code readability.

3) I would say that this piece of code could be moved to a private method and named getRandomNumber() to avoid unnecessary nesting as a passed parameter. This would make the code more readable and reusable in the future:
`new Random().nextInt(100)`

Please try to rework this class yourself, and check against my version of this class:
```suggestion
public class Lottery {
    private ColorSupplier colorSupplier;

    public Lottery(ColorSupplier colorSupplier) {
        this.colorSupplier = colorSupplier;
    }

    public Ball getRandomBall() {
        Ball ball = new Ball();
        ball.setColor(colorSupplier.getRandomColor());
        ball.setNumber(getRandomNumber(100));

        return ball;
    }
    
    private int getRandomNumber(int bound) {
        return new Random().nextInt(bound);
    }
}
```